docs/design/architecture.md

```diff
@@ -170,47 +397,89 @@ The following diagram illustrates the Kata Containers network hotplug workflow.
 ![Network Hotplug](arch-images/kata-containers-network-hotplug.png)
 
 ## Storage
```

Comment:
One thing about storage with Kata that always confused me was which type of storage is being discussed.  This whole section seems to be about just the container image filesystem and how it is mounted, but what if you pass in a VolumeMount. Does this section apply to both the container image and any mounts?